src/app/debug/table/table.component.html

```diff
@@ -33,34 +33,38 @@
 
     <div class="my-2 mx-1 px-2">
       <select data-cy-change-view-dropdown class="form-control" (change)="changeView($event)">
-        <option [style.width]="viewDropdownBoxWidth" *ngFor="let view of viewSettings.views | keyvalue" [selected]="view.key === viewSettings.currentView.name" value="{{view.key}}">{{view.key}}</option>
+        <option [style.width]="viewDropdownBoxWidth" *ngFor="let view of viewSettings.views | keyvalue" [selected]="view.key === viewSettings.currentView.name" [value]="view.key">{{view.key}}</option>
       </select>
```

Comment:
Waarom verandert hier value={{ ... }} in [value]=" ... "? Heeft dit te maken met de functionaliteit die je wilt maken of is dit ongewild veranderd?

Reply:
Functioneel veranderd dit niet, het is eerder een "good practice" om altijd deze vorm van binding aan te nemen.